sfusurge/surge_website
# 62 refactor text

## Describe your changes
Moved text content from all existing sections to a JSON file in the public folder

## Testing steps 
1. Checkout to `62-refactor-text`
2. Run the application
3. Ensure that the webpage styling and responsiveness is not affected
4. Try changing the content in the `public/content/text_content.json` file, and ensure changes persist on your local development server

## Issue ticket number and link
https://github.com/sfusurge/surge_website/issues/62

## Checklist before requesting a review
